CODE_OF_CONDUCT.md

```diff
@@ -2,76 +2,32 @@
+They have the authority and duty to delete, change, or refuse comments, code changes, documentation edits, problem reports, and other contributions that don't follow the [Code of Conduct](code-of-conduct). They can also temporarily or permanently ban any contributor for behaviors they consider inappropriate, threatening, offensive, or harmful.
 
 ## Scope
 
-This Code of Conduct applies both within project spaces and in public spaces
-when an individual is representing the project or its community. Examples of
-representing a project or community include using an official project e-mail
-address, posting via an official social media account, or acting as an appointed
-representative at an online or offline event. Representation of a project may be
-further defined and clarified by project maintainers.
+This Code of Conduct must be followed in project areas and in public when someone represents the project or its community. Examples of representation include using a project's official email, posting on official social media accounts, or being a designated representative at events, online or offline. Project maintainers may further specify what counts as representing the project.
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at ken@indicio.tech. All
-complaints will be reviewed and investigated and will result in a response that
-is deemed necessary and appropriate to the circumstances. The project team is
-obligated to maintain confidentiality with regard to the reporter of an incident.
-Further details of specific enforcement policies may be posted separately.
+If you experience or witness abusive, harassing, or otherwise unacceptable behavior, you can report it by emailing the project team at ken@indicio.tech. We will review and investigate all complaints, leading to a necessary and appropriate response based on the situation. The project team will keep the reporter’s identity confidential. We may publish more details on our enforcement policies separately.
 
-Project maintainers who do not follow or enforce the Code of Conduct in good
-faith may face temporary or permanent repercussions as determined by other
-members of the project's leadership.
+Project maintainers who fail to uphold or enforce the [Code of Conduct](code-of-conduct) properly may be subject to temporary or permanent consequences, as decided by the project’s leadership.
 
 ## Attribution
 
-This Code of Conduct is adapted from the code of conduct for the [Hyperledger
-Aries Cloud Agent -
-Python](https://github.com/hyperledger/aries-cloudagent-python) project, which
-in turn is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at [http://contributor-covenant.org/version/1/4][version]
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], [version 1.4](version).
 
 [homepage]: http://contributor-covenant.org
```

Comment:
I think maybe these link definitions need to be at the top? the links aren't working for me

Reply:
They should work once in the web.